src/Settings/Path.hs

```diff
@@ -41,6 +41,10 @@ stageDirectory = stageString
 programInplacePath :: FilePath
 programInplacePath = "inplace/bin"
 
+-- | Directory for libraries that are built "in place".
+inplaceLibPath :: FilePath
+inplaceLibPath = "inplace/lib"
+
 -- | Directory for binary wrappers, and auxiliary binaries such as @touchy@.
 programInplaceLibPath :: FilePath
```

Comment:
It may be better to rename this to `installBinPath`, and in try to maintain prefixes `inplace` and `install` throughout the codebase.